include/mips/kasan.h

```diff
@@ -25,11 +25,11 @@ static_assert(KASAN_MD_SHADOW_SIZE % SUPERPAGESIZE == 0,
   (KASAN_MD_SHADOW_START -                                                     \
    (KASAN_MD_SANITIZED_START >> KASAN_SHADOW_SCALE_SHIFT))
 
-__always_inline static inline int8_t *kasan_md_addr_to_shad(uintptr_t addr) {
+__always_inline static inline int8_t *md_addr_to_shad(uintptr_t addr) {
```

Comment:
Mmm... everything else in this file is prefixed with `KASAN` why not leave the prefix for these functions as well. We can include this header from any file (not that it would make much sense).

Reply:
I thought that it would be consistent if non-static functions begin with `kasan_`, and static ones don't. But this file is included from `mips/boot.c`, so I agree with you. Fixed in https://github.com/cahirwpz/mimiker/pull/693/commits/4a22641144052705330fe686599150bac4f922c7